### PR TITLE
Add a script to auto-detect latest build

### PR DESCRIPTION
**Pull request by Lex-2008**
Ticket: ENT-8401

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @olehermanse can review this?

File: build/latest_nightly.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# A script to output name of latest (successful) nightly build, given URL where to look and partial build name
+
+BUILDCACHE_URL="${1:?undefined}" # usually http://buildcache.cfengine.com/packages/testing-pr/
+GREP_EXPR="${2:?undefined}" # usually jenkins-master-nightly-pipeline
+
+curl --silent "$BUILDCACHE_URL" | grep "$GREP_EXPR" | sed -r 's_.*<a href="([^"/]*)/">.*_\1_' | sort -rn | while read -r build; do
+	# $build is something like jenkins-master-nightly-pipeline-962
+	# verify that it has a deb file
+	url="$BUILDCACHE_URL/$build/PACKAGES_HUB_x86_64_linux_ubuntu_16/"
```
Comment (vpodzime) [inline build/latest_nightly.sh:10]: Ubuntu 16 is soon to be gone from some builds. Let's use Ubuntu 18 or 20.

**Review by vpodzime — APPROVED** [review]
Looks good to me otherwise. Thanks!

File: build/latest_nightly.sh (hunk 1 above)
Comment (Lex-2008) [reply build/latest_nightly.sh:10]: right! https://tracker.mender.io/browse/ENT-8470 created

**Pull request merged**